### PR TITLE
Update the-obvious-hq.md...

### PR DESCRIPTION
**Pull request by tanvibhakta**
...to reflect our new space

### CODE REVIEW

File: employee-handbook/starting-at-obvious/the-obvious-hq.md [hunk 1]
```diff
@@ -2,29 +2,37 @@
...
 * the coffee machine in the kitchen is turned off
-* doors and windows are closed—be especially careful about the door to the terraces
+* doors and windows are closed—be especially careful about the doors to the balconies
 * however, don’t turn off **anyone’s** computer—they’re called _personal_ computers for a reason!
 
 If you’re the only person in the office, or if the area near the main door is unoccupied, please **keep the main door locked.**
 
 ### Timings
 
-The office is officially open Monday to Friday. We encourage you to take weekends off from work, and use the space for anything fun—screen a movie, invite friends over to play board games or party. If you do use the office over the weekend, please note that you should be extra careful about keeping the premises secure and the space clean.
```
Comment (tanvibhakta) [inline employee-handbook/starting-at-obvious/the-obvious-hq.md:24]: We should add a separate, more detailed section on using the space over the weekend, with guest policies and links to POSH policy, etc. I'll work on this in a bit.

File: employee-handbook/starting-at-obvious/the-obvious-hq.md [hunk 2]
```diff
@@ -2,29 +2,37 @@
...
+The office is officially open Monday to Friday. We encourage you to take weekends off from work. If you do use the office over the weekend, please note that you should be extra careful about keeping the premises secure and the space clean.
 
 Use the office as you please—just ensure that you follow the Entry and Exit protocol. Most people leave by 6pm, but should you need to pull an all-nighter, feel free.
 
+### Layout
+
+WIP
+
```
Comment (tanvibhakta) [inline employee-handbook/starting-at-obvious/the-obvious-hq.md:35]: I want this section to be about the various parts of the office and what they are likely to be used for.

**Comment by tanvibhakta** [comment]
I also definitely think this one long page would be much better served by creating many shorter pages, and we can work on that as well.

File: employee-handbook/starting-at-obvious/the-obvious-hq.md [hunk 3]
```diff
@@ -2,29 +2,37 @@
...
 ### Timings
 
-The office is officially open Monday to Friday. We encourage you to take weekends off from work, and use the space for anything fun—screen a movie, invite friends over to play board games or party. If you do use the office over the weekend, please note that you should be extra careful about keeping the premises secure and the space clean.
+The office is officially open Monday to Friday. We encourage you to take weekends off from work. If you do use the office over the weekend, please note that you should be extra careful about keeping the premises secure and the space clean.
 
 Use the office as you please—just ensure that you follow the Entry and Exit protocol. Most people leave by 6pm, but should you need to pull an all-nighter, feel free.
 
+### Layout
+
+WIP
```
Comment (pratul) [inline employee-handbook/starting-at-obvious/the-obvious-hq.md:34]:
```suggestion
WIP: mention the various parts of the office and what they are likely/meant to be used for.
```

**Review by pratul — CHANGES_REQUESTED** [review]

**Pull request merged**